docs/en/admins/07_LinuxUpdate.md

```diff
@@ -1,7 +1,9 @@
-# Updating on Debian 9/Ubuntu 16.04
+# Updating on Linux
 
 This tutorial demonstrates commands for updating FreshRSS. It assumes that your main FreshRSS directory is `/usr/share/FreshRSS`; If you've installed it somewhere else, substitute your path as necessary.
 
+**Note that FreshRSS has a builtin system to automatically perform an update.** It is easier to use if you don't understand the commands that follow. It is available in the web interface of your FreshRSS installation, Administration > Update.
```

Comment:
The verb `has` is used a bit much; some variation is nice while reading. ;-)

Using `>` looks a bit hackish.

```
**Note that FreshRSS contains a built-in update system.** It's easier to use if you don't understand the commands that follow. It's available through the web interface of your FreshRSS installation, Administration → Update.
```